docs/models/files.md

```diff
@@ -0,0 +1,27 @@
+Model Files
+===========
+
+A "model" is simply a shape. It can be a simple cube, it can be several cubes, it can be a truncated icosidodecahedron, or anything in between. Most models you'll see will be in [JSON] format. Models are loaded into `IModel`s by an `ICustomModelLoader` at runtime. As long as there is a corresponding model loader, all models in all formats are interchangeable.
```

Comment:
This is not true, different model loaders can have wildly different syntaxes and purposes.